Ext4Pkg/Ext4Dxe/Ext4Dxe.h

```diff
@@ -32,6 +32,7 @@
 #include "Ext4Disk.h"
 
 #define EXT4_NAME_MAX  255
+#define EFI_PATH_MAX  4096

```

Comment:
Why are we defining EFI_ constants in Ext4Dxe.h?

Reply:
Fair, could be EXT4_EFI_PATH_MAX, because it is a EFI-driver limitation based on Linux and other systems rather than  spec.